infrastructure/server/oidc_clients_spar.tf

```diff
@@ -43,9 +43,11 @@ resource "aws_cognito_user_pool_client" "test_spar_oidc_client" {
   callback_urls                                 = concat([
     "http://localhost:3000/",
     "https://oidcdebugggersecure-c6af30-dev.apps.gold.devops.gov.bc.ca/",
-    "https://nr-spar-test-frontend.apps.silver.devops.gov.bc.ca/"
+    "https://nr-spar-test-frontend.apps.silver.devops.gov.bc.ca/",
```

Comment:
Wondering do they need to remove the old one?

Reply:
Never mind, seems the ticket mentioned to keep the old one for now.

Reply:
last time when they sent me the message they want to keep the old ones, cause they haven't finish transferring to the new urls. But I'll ask them again next week just to make sure they still need the old ones.